app/src/components/inputs/autocompletes/singleAutocomplete/singleAutocomplete.jsx

```diff
@@ -161,7 +160,9 @@ export class SingleAutocomplete extends Component {
                         className={cx('input-control-btn', { 'mobile-disabled': mobileDisabled })}
                         onClick={clearSelection}
                       >
-                        <i className={cx('cross-icon')}>{Parser(CrossIcon)}</i>
+                        <i className={cx('cross-icon')}>
```

Comment:
And I suggest to use the class name such as `clear-icon` here as it is describing the action that is performed rather than name of used icon.

Reply:
Done.